sdks/java/transform-service/docker-compose/docker-compose.yml

```diff
@@ -18,6 +18,9 @@ services:
     command: -port 5001
     ports:
       - "${TRANSFORM_SERVICE_PORT}:5001"
+    depends_on:
+      - expansion-service-1
+      - expansion-service-2
```

Comment:
I'm not too familiar with how this works. Does this mean both expansion services will be built and run whenever we use the transform service?

Reply:
This is a docker compose feature. It allows specifying a dependency order between services (containers) of the Docker Compose service.

https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on

Reply:
Ok I see. Maybe later we should consider specifying the services needed for transform-service. Most use-cases won't need both java and python expansion services. Not blocking though, this LGTM

Reply:
> Most use-cases won't need both java and python expansion services

I'm hoping to use this as a generalized service that support all transforms. The main cost will be container download and storage. Once downloaded the expansion service containers can be started very quickly. Also, hoping to push changes to reduce size of the Java container in the future by removing the duplications due to using multiple shaded jars.